inventory

```diff
@@ -1,31 +1,35 @@
-bite ansible_host=bite.bitraf.no ansible_python_interpreter=/usr/bin/python3
-bitnode ansible_host=bitnode.bitraf.no
-bomba ansible_host=bomba.bitraf.no
-boxy2 ansible_host=boxy2.local
-boxy2-eof ansible_host=boxy2-eof.local
-boxy3 ansible_host=boxy3.local
-boxy3-workshop ansible_host=boxy3-workshop.local
-boxy4 ansible_host=boxy4.local
-heim ansible_host=heim.bitraf.no
-infoskjerm ansible_host=infoskjerm.local
-iot ansible_host=iot2.bitraf.no
-marvin ansible_host=2001:840:4b0b:1337:e8d9:2ff:fea1:7189
-mediawiki ansible_host=2a01:7e01:e001:e1:3a04:ccff:fe3f:7bc0
-mysql ansible_host=2a01:7e01:e001:e1:c2f7:bcff:fe31:f3cb
-nuc ansible_host=nuc.local
-p2k16 ansible_host=p2k16.bitraf.no ansible_python_interpreter=/usr/bin/python3
-p2k16-production ansible_host=p2k16-production.bitraf.no ansible_python_interpreter=/usr/bin/python3
-p2k16-staging ansible_host=p2k16-staging.bitraf.no ansible_python_interpreter=/usr/bin/python3
-radio ansible_host=radio.local
-riemann ansible_host=riemann.bitraf.no
-testskjerm ansible_host=testskjerm.local
-tv ansible_host=tv.local
-unifi ansible_host=unifi.local
-helsing ansible_host=10.13.37.59
-infoscreen-a ansible_host=infoscreen-a.local
+# NOTE(mastensg): keep list sorted and aligned.
```

Comment:
If we're cleaning up; this file can be normal a yaml file:

    aix:
      ansible_host: aix.bitraf.no
      ansible_python_interpreter: /usr/local/bin/python3
      ansible_become_method: doas

A bit(!) more verbose, but at least the format is something known.

Reply:
i like the table for easy scanning and `:sort`-ing in vim, but your suggestion would make the lines fit on standard 80-column punch cards...